server/routes/userList.js

```diff
@@ -0,0 +1,63 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../utils/dbConfig");
+
+function deleteUser(db, product) {
+    db("users")
+      .del("users.id")
+      .where("id", parseInt(product.id))
+      .then((rows) => {
+            return rows[0];
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+  
+router.get("/", async (req, res) => {
+    let page = parseInt(req.query.page) || 1;
+    await knex("users")
+      .select(
+        "users.id",
```

Comment:
same here, since you are not joining multiple tables, just use the field names without the prefix (table name)